packages/expo-updates/src/Updates.ts

```diff
@@ -90,7 +90,7 @@ export const isUsingEmbeddedAssets: boolean = ExpoUpdates.isUsingEmbeddedAssets
 
 /**
  * If `expo-updates` is enabled, this is the
- * [manifest](/workflow/expo-go#manifest) object for the update that's currently
+ * [manifest](/get-started/expo-go/#manifest) object for the update that's currently
  * running.
```

Comment:
We probably want to remove the reference to expo go here since it is technically incorrect. Much more correct to link to the types:
```
 * If `expo-updates` is enabled, this is the manifest object for the update that's currently
 * running.
 *
 * It is either a [manifest](https://docs.expo.dev/versions/latest/sdk/constants/#manifest) or a [classic manifest](https://docs.expo.dev/versions/latest/sdk/constants/#appmanifest).
 ```

(not sure how to link to relative path for these links, just linking to `/sdk/constants/#appmanifest` redirects and removes the URL fragment)

Reply:
Thanks for pointing out the technical inaccuracy. Let me update the context here and fix it and link to the correct manifest object links as well.

Reply:
Btw, the URL has to be complete relative path. For example, in this case: `/versions/latest/sdk/constants/#manifest`.